assets/style/works/works.scss

```
@@ -5,3 +5,333 @@
 @import "../globalStyle/footer";
 
 // Author: Oksana
+
+.main {
+  font-family: $font;
+  background-color: #d9d9d9; // Pasha: "D9D9D9" --> "#D9D9D9"
+
+  .main__global-container-title {
+    background-image: url(../../images/works/bg-drilling.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    min-height: 419px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-bottom: 102px;
+    @media (max-width: $laptop-l) {
+      background-position: center;
+      margin-bottom: 42px;
+    }
+
+    .main__global-container-title-text {
+      max-width: 813px;
+      width: 100%;
+      min-height: 63px;
+      @include default-p($text-light-color);
+      text-align: center;
+      @media (max-width: $tablet-l) {
+        font-size: 15px;
+        line-height: 20px;
+      }
+    }
+  }
+  .title-heading {
+    max-width: 892px;
+    margin-bottom: 31px;
+    font-family: "$font";
+    font-weight: 700;
+    font-size: 58px;
+    line-height: 55px;
+    text-align: center;
+    color: $text-light-color;
+    @media (max-width: $tablet-l) {
+      font-size: 38px;
+      line-height: 35px;
+    }
+  }
+  .main__wrapper {
+    @extend %default-wrapper;
+    .main__wrapper-common {
+      display: flex;
+      @media (max-width: $pc-m) {
+        margin-left: 20px;
+        margin-right: 20px;
+        width: 100%;
+
+        @media (max-width: $laptop-l) {
+          min-height: 439px;
+          width: 100%;
+          align-items: center;
+          justify-content: space-between;
+        }
+      }
+      .main__wrapper-common-picture {
+        margin-right: 47px;
+        @media (max-width: $laptop-l) {
+          width: 30%;
+        }
+        @media (max-width: $phone-l) {
+          display: none;
+        }
+      }
+      .main__wrapper-common-text-paragrph {
+        @include default-p($text-blue-color);
+        margin-top: 20px;
+      }
+    }
+    .main__wrapper-minerals-title {
+      margin-bottom: 20px;
+      margin-top: 40px;
+      font-weight: 700;
+      font-size: 32px;
+      line-height: 28px;
+      color: $text-black-color;
+      @media (max-width: $pc-m) {
+        margin-left: 20px;
+        margin-right: 20px;
+      }
+      @media (max-width: $phone-l) {
+        margin-top: 20px;
+      }
+    }
+    .main__wrapper-common-text-paragrph {
+      @include default-p($text-blue-color);
+      margin-bottom: 20px;
+      @media (max-width: $pc-m) {
+        margin-left: 20px;
+        margin-right: 20px;
+      }
+    }
+  }
+  .main__wrapper-common-best {
+    max-width: 1420px;
```

На самом деле не обязательно было прописывать явно ширину 1420px. Можно было оставить дефолтную (1600px), просто поставить для элементов
justify-content: flex-start;
Чтобы они начинались слева.
```suggestion
    @extend %default-wrapper;
    justify-content: flex-start;
```

---

Adaptive
[Тут](https://i.ibb.co/tMDYphR/2023-04-19-101218347.png) не помешал бы
column-gap: 10px;